lib/modules/hover.js

```diff
@@ -309,6 +309,12 @@ addModule('hover', function(module, moduleID) {
 				.off('mouseleave', this.cancelShow);
 		},
 		startShowTimer: function() {
+			this.mouseEvent = 'enter';
+
+			if (this.visible && this.mouseEvent === 'enter') {
```

Comment:
Whoops! Must have had tunnel vision. This.visible should be sufficient, I'll take another look.